headers/storagemanager.h

```diff
@@ -18,6 +18,7 @@
 
 #include <atomic>
 
+
```

Comment:
Should not be adding spaces here, check your "git diff" before you commit to find these things.